partner_preview_scripts/encore_parser.py

```diff
@@ -0,0 +1,443 @@
+#!/usr/bin/env python3
+"""Parser for data submitted from the Encore team."""
+
+import argparse
+import json
+import logging
+import os
+import sys
+from functools import reduce
+
+
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import col, struct, regexp_replace, udf, lit, expr, split, explode, array
+from pyspark.sql.types import StringType, StructType, StructField, ArrayType, FloatType
+from pyspark.sql.dataframe import DataFrame
+
+from common.evidence import initialize_sparksession, write_evidence_strings, GenerateDiseaseCellLines
+
+class EncoreEvidenceGenerator:
```

Comment:
```suggestion
from common.evidence import initialize_sparksession, write_evidence_strings, GenerateDiseaseCellLines


class EncoreEvidenceGenerator:
```

(Adding an extra blank line before class definition.)